.travis.yml

```diff
@@ -1,11 +1,11 @@
-language: c
+language: python
 
 python:
     - "2.7"
 
 os:
     - linux
-    - osx
+    #- osx
```

Comment:
Travis currently checks if the build compiles on OSX despite that we can't run the tests.
Please don't comment out this line.